### PR TITLE
Tooltip component

### PR DESCRIPTION
**Pull request by mperrotti**
#### Related issues
Fixes https://meetup.atlassian.net/browse/MUP-19431

#### Description
Added a `Tooltip` component that gets it's positioning from a new wrapper component, `FloatingPosition`.
There's a [follow-up ticket](https://meetup.atlassian.net/browse/SDS-587) to replace the positioning code in `Dropdown.jsx` by wrapping the dropdown content in `<FloatingPosition />`

#### Screenshots (if applicable)
screen shot 2018-04-17 at 8 54 12 pm

### CODE REVIEW

File: src/utils/components/FloatingPosition.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+					})
+				}
+			</ConditionalWrap>
+		);
+	}
+
+}
+
+FloatingPosition.propTypes = {
+	getTrigger: PropTypes.func
```
Comment (akdetrick) [inline src/utils/components/FloatingPosition.jsx]: `isRequired`?

File: assets/scss/components/_dropdown.scss [hunk 2]
```diff
@@ -22,17 +22,12 @@ $spaceFromTrigger: $space-3 + $triangleSize;
...
-.dropdown-content {
+.dropdown-bubble {
 	border: 1px solid $C_border;
 	border-radius: $defaultRadius;
 	background-color: $C_contentBG;
 	box-shadow: $dropdownShadow;
 	box-sizing: border-box;
-	margin-top: $triangleSize;
-	position: absolute;
-	transition: left $duration--short $easing--standard,
```
[the remark this answers is not in the document]
Comment (mperrotti) [reply assets/scss/components/_dropdown.scss]: Yes, it was causing issues on mount. It was only there to make the re-positioning smoother when scrolling and resizing.
I can either:
- fix the mounting issues now
- fix the mounting issues in a follow-up
- leave it as-is

I vote "fix the mounting issues in a follow-up", but I can do them now if you think it's a blocker

File: assets/scss/components/_tooltip.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.dropdown-content--tooltip {
```
[the remark this answers is not in the document]
Comment (mperrotti) [reply assets/scss/components/_tooltip.scss]: I'll file a follow-up ticket

File: src/interactive/Tooltip.jsx [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+		this.openContent = this.openContent.bind(this);
+
+		this.state = {
+			isActive: props.isActive || false,
+		};
+
+	}
+
+	closeContent(e) {
+		if (this.props.manualClose && this.props.isActive) {
```
[the remark this answers is not in the document]
Comment (mperrotti) [reply src/interactive/Tooltip.jsx]: I added it at first, but deleted later and missed these lines

**Pull request merged**